site/content/blog/2023-08-25-community-tips-and-tricks/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Community - Tips and Tricks"
+summary: >
+  News about a community area to contribute ZAP usage tips and tricks.
+images:
+- https://www.zaproxy.org/blog/2023-08-25-community-tips-and-tricks/images/zapbot.TandT.gif
+addSocialPreview: true
+type: post
+date: "2023-08-25"
```

Comment:
No tags?
We usually include at least "blog"

Reply:
Hmmm not sure how I missed that, I'll get some added.